config/secrets.yml.example

```diff
@@ -30,6 +41,16 @@ test:
         hwIDAQAB
         -----END PUBLIC KEY-----
       encryption_private_key: 'RvGHVZ/kvzUAA5Z3t68+FNhuMCJxkzv+'
+  content:
+      abl_url: https://raw.githubusercontent.com/openstax/content-manager-approved-books/master/approved-book-list.json
+      archive_path: apps/archive
+      bucket_name: <%= ENV['OPENSTAX_CONTENT_BUCKET_NAME'] || 'not-a-real-bucket' %>
+      domain: openstax.org
+      s3_region: <%= ENV['OPENSTAX_CONTENT_S3_REGION'] || 'us-east-1' %>
+      s3_access_key_id: <%= ENV['OPENSTAX_CONTENT_S3_ACCESS_KEY_ID'] || 'not-a-real-key' %>
+      s3_secret_access_key: <%= ENV['OPENSTAX_CONTENT_S3_SECRET_ACCESS_KEY'] || 'not-a-real-key' %>
+      rex_release_url: <%= ENV['REX_RELEASE_URL'] || 'https://dev.openstax.org/rex/release.json' %>
+      rex_config_url: <%= ENV['REX_CONFIG_URL'] || 'https://dev.openstax.org/rex/config.json' %>
 
 production:
```

Comment:
do we need to add the content config block to production as well?

Reply:
I'm not sure if the deploy for this just copies the example to make the secrets.yml or if it's manual and the config sticks around, either way it's probably helpful to add it to that block, I'll update it